src/routes/recipes.js

```diff
@@ -48,8 +52,9 @@ function RecipeItem(props) {
 }
 
 function RecipeList(props) {
-  const listItems = props.recipes.map((recipe) =>
-    <RecipeItem key={recipe.name} recipe={recipe}/>
+  let recipeSection_keys = Object.keys(props.recipes);
```

Comment:
It's a little more typical to use camelcase in React/JS (eg. `recipeSectionKeys`). I also was not quite sure based on the naming what `recipeSection_keys` represents since its items are referred to as `recipe` and `recipeId` below. Are these recipe IDs?

```suggestion
  let recipeIds = Object.keys(props.recipes);
```